.golangci.yml

```diff
@@ -29,7 +29,9 @@ linters:
     # - gochecknoinits
     - gocritic
     - gocyclo
+    - goconst
```

Comment:
I'm wondering if this check is a subset of other checks, since it didn't flag anything new. Does it flag anything in any of the other Go-heavy repos?

Reply:
It did send 2 warnings for strongswan code which is now removed. See @skitt's comment above. So this check was not covered before.

Reply:
Ah, okay that's good! Agreed that means it's good to enable. :heavy_check_mark: